pkg/minikube/command/ssh_runner.go

```diff
@@ -189,6 +196,49 @@ func (s *SSHRunner) Copy(f assets.CopyableFile) error {
 	return g.Wait()
 }
 
+func (s *SSHRunner) fileExistsInVM(f assets.CopyableFile, dst string) bool {
+	sess, err := s.c.NewSession()
+	if err != nil {
+		return false
+	}
+
+	// check if sizes of the two files are the same
+	srcSize := f.GetLength()
+	size := fmt.Sprintf("ls -l %s | cut -d \" \" -f5", dst)
+	out, err := sess.CombinedOutput(size)
+	if err != nil {
+		return false
+	}
+	dstSize, err := strconv.Atoi(strings.Trim(string(out), "\n"))
+	if err != nil {
+		return false
+	}
+	if srcSize != dstSize {
+		return false
+	}
+
+	sess, err = s.c.NewSession()
+	if err != nil {
+		return false
+	}
+	// ensure src file hasn't been modified since dst was copied over
+	srcModTime := f.GetModTime()
+	stat := "stat -c %Y" + fmt.Sprintf(" %s", dst)
```

Comment:
Would it be possible to combine the two ssh commands into one? I think there is a `stat` command that will return both, or you could combine two stat commands in a single session: `stat -c <something> <path> && stat -c %Y <path>`

Reply:
done